Dockerfile

```diff
@@ -1,19 +1,28 @@
-FROM alpine
+FROM debian:11 as builder
+ARG PDNS_VERSION=4.7.3
+
+WORKDIR /build
+RUN apt update && \
+    apt install -y curl bzip2 g++ python3-venv libtool make pkg-config \
+    libboost-all-dev libssl-dev libluajit-5.1-dev libcurl4-openssl-dev libsqlite3-dev
+RUN curl -sL https://downloads.powerdns.com/releases/pdns-$PDNS_VERSION.tar.bz2 | tar -jx
+WORKDIR /build/pdns-$PDNS_VERSION
+RUN ./configure --with-modules='bind gsqlite3' && \
+    make -j $(nproc) && \
+    make install
+
+FROM debian:11-slim
```

Comment:
If we use debian for building, we use debian for running it, for those who want to stay away from the dragons shan't mix libc implementations

Reply:
Agree :+1: